### ISSUE TITLE
3.0.0 release not Java 8 compatible

### ISSUE THREAD

**Issue by johanandren** [akka/alpakka#2680]
Reported in the forums: https://discuss.lightbend.com/t/the-alpakkas-3-0-and-the-alpakka-kafka-2-1-are-loose/8246

Needs further investigation, very curious.

**Comment by johanandren** [comment]
The problem is the release on travis-ci running on Java 11 making any Java sources in the modules compile with that (major version: 55), we need to fix the release-scripts and release a fixed 3.0.1 that can be used on Java 8

**Comment by ennru** [comment]
Fixed with release 3.0.1

**Comment by jphelp32** [comment]
@johanandren There seems to be a regression of this issue in alpakka 6.0.1. In particular it seems to affect only the Enum classes. I've found these so far:
```
akka.stream.alpakka.csv.javadsl.CsvQuotingStyle
akka.stream.alpakka.xml.ParseEventMarker
```
e.g.
```
% javap -v akka.stream.alpakka.xml.ParseEventMarker | grep "major version"
  major version: 55
```
